docs/streams/upgrade-guide.html

```diff
@@ -98,6 +98,9 @@ <h3><a id="streams_api_changes_300" href="#streams_api_changes_300">Streams API
     </p>
     <ul>
         <li> <code>WindowBytesStoreSupplier#segments</code>: deprecated in Kafka 2.1.0 (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-319%3A+Replace+segments+with+segmentInterval+in+WindowBytesStoreSupplier">KIP-319</a>).</li>
+        <li> Overloaded <code>KafkaStreams#close</code>: deprecated in Kafka 2.1.0 (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-319%3A+Replace+segments+with+segmentInterval+in+WindowBytesStoreSupplier">KIP-319</a>).</li>
```

Comment:
Missing note for removed methods on KeyQueryMetadata and StreamsMetadataState

Reply:
KeyQueryMetadata: ah yes, will add.
StreamsMetadataState: it is the internal class.

Reply:
EDIT: deprecated APIs in KeyQueryMetadata are part of KIP-648 (2.7) so should not be removed, will add back.